### PR TITLE
[release notes] Add new frontmatter for the indexer

### PR DESCRIPTION
**Pull request by timonrey**
closes https://github.com/commercetools/commercetools-docs/issues/4999

Smoke test example with multiple types:
https://docskit-tt-add-frontmatter-for-release-note-indexer.commercetools.vercel.app/docs-smoke-test/releases

### CODE REVIEW

File: websites/documentation/src/content/writing/release-notes.mdx [hunk 1]
```diff
@@ -65,7 +65,13 @@ If there is no more, remove the marker above, too.
...
+- `type` (string | string[], optional): Must be either `feature` (for "New Feature"), `fix` (for "Resolved Issue") or `enhancement` (for "Enhancement")
 - `topics` (array of string, optional): YAML array of free to choose text entries. The Filter UI may be constrained to pre-defined topics.
 - `published` (boolean, optional, defaults to `true`): `false` prevents rendering of the release in the site. **Should not be used in the main documentation**, reserved for atypical situations. Not included in the template.
 - `slug` (string, optional, defaults to `$date-$title` (slugified)): optional, allows overriding the autogenerated URL slug. **Should not be used in the main documentation**, reserved for migrating legacy docs.
+- `product` (string[], optional): optional, used in the release notes search to filter by product. If set, it overrides the `product` defined in the website `gatsby-configure.mjs`. Please note that in case an array is defined, just the first value is used for filtering purposes.
+- `productArea`: Option, used in the release notes search to filter by product area. If set, it overrides the `title` defined in the website `gatsby-configure.mjs`.
```
Comment (FFawzy) [inline websites/documentation/src/content/writing/release-notes.mdx:73]: @timonrey lets adjust the docs accordingly to the latest changes .. and remove the console logs that were missed

**Comment by timonrey** [comment]
@zbalek @melindafekete 
It's basically the same as before. Only that you can now define multiple types for the release note...and they will be displayed next to each other.

**Comment by zbalek** [comment]
Makes sense from my perspective, can you please check that the colors-types look as follows:
image

Also, when it comes to marking the products/product areas on the areas, please see the designs:
https://www.figma.com/design/GqhHFUcEaMxBXnMM9TXdaZ/%F0%9F%9F%A2-Feedback-Board?node-id=4743%3A15885&t=Dx4AbVlZ32DcFVtq-1

image

**Comment by timonrey** [comment]
@FFawzy I can include the deprecation type in this PR already, but I would handle the product areas separately if that's fine.

**Comment by melindafekete** [comment]
Looks all good from my side. Thank you!

**Review by gabriele-ct — APPROVED** [review]
nice 👍

**Pull request merged**